airbyte-integrations/connectors/source-monday/acceptance-test-config.yml

```diff
@@ -1,34 +1,51 @@
 # See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
 # for more information about how to configure these tests
 connector_image: airbyte/source-monday:dev
-tests:
+acceptance_tests:
   spec:
-    - spec_path: "source_monday/spec.json"
+    tests:
+      - spec_path: "source_monday/spec.json"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "secrets/config_api_token.json"
-      status: "succeed"
-    - config_path: "secrets/config_oauth.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
-    - config_path: "integration_tests/invalid_config_oauth.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config_api_token.json"
+        status: "succeed"
+      - config_path: "secrets/config_oauth.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
+      - config_path: "integration_tests/invalid_config_oauth.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
-    - config_path: "secrets/config_api_token.json"
-    - config_path: "secrets/config_oauth.json"
+    tests:
+      - config_path: "secrets/config_api_token.json"
+        backward_compatibility_tests_config:
+          disable_for_version: "0.2.0"
+      - config_path: "secrets/config_oauth.json"
+        backward_compatibility_tests_config:
+          disable_for_version: "0.2.0"
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      empty_streams: ["updates"]
-    - config_path: "secrets/config_api_token.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      empty_streams: ["updates"]
-    - config_path: "secrets/config_oauth.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      empty_streams: ["updates"]
+    tests:
+      - config_path: "secrets/config_api_token.json"
+        configured_catalog_path: "integration_tests/configured_catalog.json"
```

Comment:
if I'm not mistaken, you should omit this for high strictness level

Reply:
Done